plugins/lookup/conjur_variable.py

```diff
@@ -203,12 +210,24 @@ def _fetch_conjur_variable(conjur_variable, token, conjur_url, account, validate
     return {}
 
 
+def _store_in_file(value):
+    file_name = ''.join(choice(string.ascii_uppercase + string.digits) for i in range(12))
+    file_path = os.path.join("/dev/shm", file_name)
+    with open(file_path, 'w+') as file:
+        file.write(value[0])
+        os.chmod(file_path, stat.S_IRUSR | stat.S_IWUSR)
+
+        return [file_path]
```

Comment:
This return should happen as part of the function context, not the `with` context so it should be indented 1 lower level

Reply:
This piece of code is remove in next commit thanks to  tempfile.NamedTemporaryFile.